pom.xml

```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <!-- TODO: replace with basepom --> 
-    <!-- <parent>
+    <parent>
       <groupId>org.jboss</groupId>
       <artifactId>jboss-parent</artifactId>
       <version>28</version>
       <relativePath />
-    </parent> -->
-    
+    </parent> 
+<!--
```

Comment:
we want basepom based Teiid, this seems wrong?? looks like someone already did basepom changes